src/components/SpiceItem.js

```diff
@@ -3,17 +3,18 @@ import React from 'react'
 class SpiceItem extends React.Component {
 
   render() {
+
     return (
       <div className="spice-item">
-        <img src={null /* replace me with an image url */} alt={"replace me with a TITLE"} />
+        <img src={this.props.spice["image"]} alt={`${this.props.spice["title"]}`} />
         <div className="details">
-          <h2>{"replace me with a TITLE"}</h2>
-          <p>{"replace me with a DESCRIPTION"}</p>
-          <em>{"replace me with NOTES"}</em>
+          <h2>{this.props.spice["title"]}</h2>
+          <p>{this.props.spice["description"]}</p>
+          <em>{this.props.spice["notes"]}</em>
```

Comment:
Nice! You can also use the dot notation here to access keys from the spice object, like this:

```jsx
<img src={this.props.spice.image} alt={this.props.spice.title} />
<div className="details">
  <h2>{this.props.spice.title}</h2>
  <p>{this.props.spice.description}</p>
  <em>{this.props.spice.notes}</em>
</div>
```